components/automatic-points-table/commons/automatic-points-table.lua

```diff
@@ -298,4 +300,82 @@ function AutomaticPointsTable:addPositionData(pointsData)
 	)
 end
 
+function AutomaticPointsTable:generatePointsTable(pointsData, tournaments)
+local columnCount = Array.reduce(tournaments, function(count, t)
+			return count + (t.shouldDeductionsBeVisible and 2 or 1)
+		end, 0)

```

Comment:
```suggestion
		end)

```

Reply:
Doing this initializes the aggregate to tournaments[1] which is a table value, so it produces this error
![image](https://user-images.githubusercontent.com/28851891/150774700-d78925b3-3b6c-434b-be0e-297cce1e2903.png)

Reply:
Ah yea, you're right